CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### Grafana Mimir
 
+* [ENHANCEMENT] Query-scheduler: add `cortex_query_scheduler_enqueue_duration_seconds` metric that records the time taken to enqueue or reject a query request. #5879
+* [ENHANCEMENT] Query-frontend: add `cortex_query_frontend_enqueue_duration_seconds` metric that records the time taken to enqueue or reject a query request when not using the query-scheduler. #5879
+* [ENHANCEMENT] Expose `/sync/mutex/wait/total:seconds` Go runtime metric as `go_sync_mutex_wait_total_seconds_total` from all components. #5879
```

Comment:
I'm guessing `go_sync_mutex_wait_total_seconds_total` was named such for a reason, but "total_seconds_total" sticks out. Maybe `go_sync_mutex_wait_seconds_total`?

Reply:
Unfortunately this name is created automatically by the Golang Prometheus library, and I don't believe we can override it: https://github.com/grafana/mimir/blob/c5743267204284976dfb1a8c490977a984af03bc/vendor/github.com/prometheus/client_golang/prometheus/internal/go_runtime_metrics.go#L39